extensions/geode-modules-session/build.gradle

```diff
@@ -37,9 +37,9 @@ dependencies {
   }
   integrationTestCompile('commons-io:commons-io:' + project.'commons-io.version')
   integrationTestCompile('javax.servlet:javax.servlet-api:' + project.'javax.servlet-api.version')
-  integrationTestCompile('junit:junit:4.12')
+  integrationTestCompile('junit:junit:' + project.'junit.version')
```

Comment:
I thought groovy allowed something like `integrationTestCompile("junit:junit:${project.'junit.version'}")`?  But I find consistency `>` groovy conventions.

Reply:
I would actually prefer the string interpolation, since I think that is more readable.  But I also believe that consistency is the more important thing here.

Also, in the longer term, we'll do away with all of this `+ project.'lib.version'` stuff.  Gradle 5 has dependency constraints ([Gradle docs here](https://docs.gradle.org/current/userguide/managing_transitive_dependencies.html)), but they're currently not playing nicely with the `maven-publish` plugin we use.  Alternatively, there's Spring's Gradle Dependency Management plugin ([here](https://github.com/spring-gradle-plugins/dependency-management-plugin)), but I haven't had the chance to look meaningfully at that yet.